iceberg-handler/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.hive</groupId>
+        <artifactId>hive</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>iceberg-handler</artifactId>
+    <packaging>jar</packaging>
+    <name>Hive Iceberg Handler</name>
+
+    <properties>
+        <hive.path.to.root>..</hive.path.to.root>
+        <iceberg-api.version>0.11.0</iceberg-api.version>
```

Comment:
I would prefer to have this in the top level pom. `iceberg.version`
Why is this here?

Reply:
Sure, will move it to the top level pom

Reply:
Moved it